std_msgs/Empty request
---
Location pose
